### DISCUSSION TITLE
Help with ventoy_grub.cfg

### DISCUSSION THREAD

**Discussion by vadlike** [ventoy/Ventoy#1202] [Q&A]
How can i add this code

#########
menuentry "name"  {
}
#########
on two configs?

1# cfg
#####################
f [ "$grub_platform" == "efi" ]; then
search --file --no-floppy --set=root /EFI/BOOT/grubfm
chainloader /efi/boot/grubfm${VTOY_EFI_ARCH}.efi
  boot
  fi
  elif [ "${grub_platform}" == "pc" ]; then
search --file --no-floppy --set=root /AIO/grubfm/grubfm
    set _path="/AIO/grubfm/"
  linux ${_path}/loadfm
    initrd ${_path}/grubfm.iso
  boot
    }
####################
2# cfg
####################
if [ "$grub_platform" == "efi" ]; then
search -n -f "/AIO/grub/grub.cfg" -s root
chainloader /efi/boot/boot${VTOY_EFI_ARCH}.efi
  boot
  fi
  elif [ "${grub_platform}" == "pc" ]; then
search --file --no-floppy --set=root /AIO/grub/i386-pc/core.img
multiboot /AIO/grub/i386-pc/core.img
  boot
    }
#####################

**Comment by steve6375** [comment]
Use F6 
https://ventoy.net/en/plugin_grubmenu.html

**Reply by vadlike** [reply]
> Use F6 https://ventoy.net/en/plugin_grubmenu.html

i know that. but i need to correct my configs. i need to add this menuentry "name" {
} to my configs

**Comment by steve6375** [comment]
Then just put your code inside the {     } braces
something like this...

```
menuentry "name" {
if [ "$grub_platform" == "efi" ]; then
search --file --no-floppy --set=root /EFI/BOOT/grubfm
chainloader /efi/boot/grubfm${VTOY_EFI_ARCH}.efi
boot
elif [ "${grub_platform}" == "pc" ]; then
search --file --no-floppy --set=root /AIO/grubfm/grubfm
set _path="/AIO/grubfm/"
linux ${_path}/loadfm
initrd ${_path}/grubfm.iso
boot
fi

}

```

**Reply by vadlike** [reply]
Thank you so much. its working. ;) i know a little grub2. but i dont know what means elif, fi, if

**Reply by steve6375** [reply]
elseif

**Reply by steve6375** [reply]
if [  test ]; then xxxx; fi
if [ test ]; then xxxx; elif  yyyy; fi
if is start of if code and fi is end of the if code
Maybe you need my eBooks on Ventoy and a1ive grub2 file manager ;-) https://payhip.com/SteveSi
